### ISSUE TITLE
Clients needs to be registered w/ both satellite and external capsule for client upgrade

### ISSUE THREAD

**Issue by san7ket** [SatelliteQE/satellite6-upgrade#24]
Need to add support that client should be registered w/ both satellite and capsule for client upgrade. Currently, I think clients only registered w/ satellite.
Issue: https://github.com/SatelliteQE/automation-tools/issues/518

**Comment by san7ket** [comment]
I have a question regarding this, looking at how we are implemented the upgrade process.
Product is satellite or capsule or client or longrun.
    If product is satellite then upgrade only satellite
    If product is capsule then upgrade satellite and capsule
    If product is client then upgrade satellite and client
    If product is longrun then upgrade satellite, capsule and client
    If product is n-1 then upgrades only satellite by keeping capsule at last
    z-stream released version

Should this go under Long-run? @sghai @jyejare

**Comment by sghai** [comment]
Yeah.. longrun would be the way to go.

**Comment by jyejare** [comment]
Correct, As Capsule has to be upgraded first before upgrading the clients, Longrun can be effective here.
